tests/nocover/test_compat.py

```diff
@@ -106,13 +108,13 @@ def test_to_bytes_in_big_endian_order(x, y):
     assert int_to_bytes(x, 8) <= int_to_bytes(y, 8)
 
 
-@pytest.mark.skipif(sys.version_info[0] != 3 or sys.version_info[:2] == (3, 5),
+@pytest.mark.skipif(not hasattr(inspect, 'getfullargspec'),
                     reason='getfullargspec was deprecated, so we wrap it')
```

Comment:
I think either reason or new check is not correct. `getfullargsspect` was deprecated only in Python 3.5, but in 3.6 it was restored as normal function. However, this function is present in all 3.x series.

Reply:
Good catch - the reason is now out of date; this works on all Python 3.x.